plugins/tracker-resources/src/components/NewIssueHeader.svelte

```diff
@@ -39,27 +40,42 @@
       closed = true
     })
   }
+  const dropdownItems: SelectPopupValueType[] = [
+    {
```

Comment:
We need both CreateIssue and CreateProject in dropdown I suppose.

Reply:
I disagree. I don't think we need to double the same action when we already have it right there